test/Microsoft.Azure.WebJobs.Host.EndToEndTests/SingletonEndToEndTests.cs

```diff
@@ -272,24 +273,22 @@ public async Task SingletonFunction_StorageAccountOverride()
 
             // make sure the lease blob was only created in the secondary account
             await VerifyLeaseDoesNotExistAsync(method, SingletonScope.Function, null);
-            await VerifyLeaseState(method, SingletonScope.Function, null, LeaseState.Available, LeaseStatus.Unlocked, directory: _secondaryLockDirectory);
+            await VerifyLeaseState(method, SingletonScope.Function, null, LeaseState.Available, LeaseStatus.Unlocked, directory: _secondaryLockDirectoryContainerClient);
         }
 
         // Allow a host to override container resolution. 
-        class CustomLockManager : StorageBaseDistributedLockManager
+        class CustomLockManager : BlobLeaseDistributedLockManager
```

Comment:
Previously with the base StorageBaseDistributedLockManager public, the override pattern was to subclass and override a single method. Now the only way to do this is to fully implement IDistributedLockManager? In these tests you're using internals access to just simplify that implementation. I can't remember why the abstract GetContainer pattern was added - may not be important anymore.